test/unit/setup/helpers.js

```diff
@@ -15,6 +15,14 @@ var clearFixtures = function () {
   $body.empty();
 };
 
+before(function() {
+  this.checkProperties = function(block, blacklist) {
+    blacklist = blacklist ? blacklist.push('cid') : 'cid'
+    var props = _.partial(_.omit, _, blacklist);
+    block.call(this, props);
+  }
```

Comment:
Was missing require directory three for jscs and jshint. Look this lines [jscs src](https://github.com/marionettejs/backbone.marionette/pull/2524/files#diff-35b4a816e0441e6a375cd925af50752cR207) and [jslint](https://github.com/marionettejs/backbone.marionette/pull/2524/files#diff-35b4a816e0441e6a375cd925af50752cR189). Thanks